src/main/java/com/sarahhand/fractals/view/FractalsUI.java

```diff
@@ -44,28 +45,48 @@
 
 	private JFrame frame;
 	private Dimension frameDimension;
+
 	private FractalEventHandlerFactory eventHandlerFactory;
-	private FractalEventHandler events;
-	
+	private FractalEventHandler mandelbrotEvents;
+	private FractalEventHandler juliaEvents;
+	private FractalEventHandler currentEvents;
+
 	private ImageIcon image;
 	private JLabel imageLabel;
 
 	private JMenuBar menuBar;
 
 	private JMenu fileMenu;
 	private JMenuItem saveMenuItem;
-	private JMenuItem loadMenuItem;
+
+	private JMenu loadMenu;
+	private JMenuItem loadMandelbrotMenuItem;
+	private JMenuItem loadJuliaMenuItem;
+
 	private JMenuItem saveImageMenuItem;
 	private JMenuItem exitMenuItem;
 
 	private JMenu optionsMenu;
 	private JMenuItem createColorPaletteMenuItem;
+	private JMenuItem setJuliaConstantsMenuItem;
 
 	private JMenu selectColorSchemeMenu;
 
+	private JMenu fractalMenu;
+	private JRadioButtonMenuItem mandelbrotSetMenuItem;
+	private JRadioButtonMenuItem juliaSetMenuItem;
+
 	private final int FRAME_WIDTH = 800;
 	private final int FRAME_HEIGHT = 600;
 
+	public FractalEventHandler getJuliaEvents() {
+		return juliaEvents;
+	}
+
+	private FractalsUI getThis() {
```

Comment:
The same thing occurs with this method as the other one. It is used in the ActionListeners.